sql/rpl_master_info_file.hh

```diff
@@ -0,0 +1,622 @@
+/*
+  Copyright (c) 2025 MariaDB
+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; version 2 of the License.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License along
+  with this program; if not, write to the Free Software Foundation, Inc.,
+  51 Franklin St, Fifth Floor, Boston, MA 02110-1335 USA.
+*/
+
+#include "rpl_info_file.hh"
+#include <unordered_map> // Type of @ref MasterInfoFile::FIELDS_MAP
+#include <string_view>   // Key type of @ref MasterInfoFile::FIELDS_MAP
+#include <unordered_set> // Parameter type of ChangeMaster::set_defaults() //?
```

Comment:
Is this needed? I only saw `seen` of that type, and I'm not certain is is effectively used anymore (perhaps it was needed in an earlier version?)

Reply:
Aye, that comment was a leftover from an earlier MDEV-28302 design that I expect to replace.